### PR TITLE
Fix bug of 'g g' not scrolling to top

### LINKED ISSUE #2874: gg should scroll to the top
Hitting 'g g' currently sets the cursor to the right place (at the top), but it doesn't scroll there anymore, which is bad.

### PR DESCRIPTION
**Pull request by yuqijin**
### Reasons for making this change

fix gg shortcut

### Related issues

fix #2874 

### Screenshots

2874-gg

### Checklist

* [ ] I've added a screenshot of the changes, if this is a frontend change
* [ ] I've added and/or updated tests, if this is a backend change
* [ ] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [ ] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/components/worksheets/WorksheetItemList.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class WorksheetItemList extends React.Component {
                 $('body')
                     .stop(true)
                     .animate({ scrollTop: 0 }, 'fast');
-                this.props.setFocus(-1, 0);
+                this.props.setFocus(0, 0);
```
Comment (percyliang) [inline frontend/src/components/worksheets/WorksheetItemList.js]: We actually want the cursor to be at the "-1" position though, before any bundles.

**Review by yipenghe — COMMENTED** [review]
The setFocus should be -1 as described in the comment near line 130. The root cause is here:
https://github.com/codalab/codalab-worksheets/blob/master/frontend/src/components/worksheets/Worksheet/Worksheet.js#L666

The node here will be null (you can console.log the node) when we set the focus above the -1 item, since there is no item with id `codalab-worksheet-item--1`. A possible fix will to manually give the -1 item the id  codalab-worksheet-item--1 or some other special id, and find the item using that id when the passed in index argument is -1

File: frontend/src/__tests__/__snapshots__/Worksheet.test.js.snap [hunk 2]
```diff
@@ -542,6 +542,7 @@ Object {
             >
               <div
                 class="Worksheet-worksheetDummyHeader-4"
+                id="worksheet_header"
```
Comment (yipenghe) [inline frontend/src/__tests__/__snapshots__/Worksheet.test.js.snap:545]: let's use worksheet_dummy_header for consistency

**Review by yipenghe — APPROVED** [review]
Works well!

**Pull request merged**